Mustache/src/html/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="utf-8">
+    <title>Agentes</title>
+    <link rel="stylesheet" href="main.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script type="text/javascript" src="../js/main.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/3.0.0/mustache.min.js"></script>
+</head>
+<body class="page">
+    <header class="header">
+        <img class="header__logo" src="/images/logo.png">
+        <div class="menu">
+            <nav class="menu__nav">
+                <ul class="menu__items">
+                    <li class="menu__item"><a class="menu__item--select menu__item--first " href="#" onclick="showAll()">Directorio de Agentes</a></li>

```

Comment:
Don't mix JS in HTML using onclick="method()". Use event handlers instead.
```
// Assign a function to deal with the onclick event:
document.getElementById('id').onclick = doSomething;

// or use an anonymous function:
document.getElementById('id').onclick = function() {
    // do magic
};
```

https://stackoverflow.com/questions/5871640/why-is-using-onclick-in-html-a-bad-practice